languages/elixir/exercises/concept/binary-matching/.docs/introduction.md

```diff
@@ -0,0 +1,52 @@
+Elixir provides an elegant syntax working with binary data as we have seen with the `<<>>` special form provided for working with _bitstrings_.
+
+## Binary data
+
+The binary type is a specialization on the bitstring type. Where bitstrings could be of any length (any number of [bits][wiki-bit]), binaries are where the number of [bits][wiki-bit] can be evenly divided by 8. That is, when working with binaries, we often think of things in terms of [bytes][wiki-bytes] (8 [bits][wiki-bit]). A [byte][wiki-bytes] can represent integer numbers from `0 - 255`. It is common to work with [bytes][wiki-byte] values in [hexadecimal][wiki-hexadecimal], `0x00 - 0xFF`.
```

Comment:
> integer numbers from `0 - 255`.

Is this grammatically correct? Sounds to me like it should be more like "integer numbers from `0` to `255`" or "integer numbers from the range `0 - 255`", but I am not sure.

Reply:
Yours flows better, I like it. 👍